sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -31,6 +31,9 @@ For any DPC++-related commit, the `[SYCL]` tag should be present in the
 commit message title. To a reasonable extent, additional tags can be used
 to signify the component changed, e.g.: `[PI]`, `[CUDA]`, `[Doc]`.
 
+## Using \<iostream\> 
+
+According to [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden), the use `#include <iostream>` is forbidden in library files. Thereby it's suggested to use the `<sycl/detail/iostream_proxy.hpp>` header instead, as it has the functionality of `<iostream>` without it's static constructor.
```

Comment:
Please try and keep with the style of the rest of the document, i.e. try and keep lines under the 80 character cap as much as possible.
Please also add an empty line after this.

I would probably reword the second sentence a little, to something like; "Instead, the `sycl/detail/iostream_proxy.hpp` header offers the functionality of `<iostream>` without its static constructor. This header should be used in place of `<iostream>` in DPC++ headers and runtime library files."

Reply:
Ohh. My bad, didn't know there was a 80 character cap. Will apply the other suggestions

Reply:
The section still seems to be on one line.

Reply:
By one line I thought you meant the characters within a sentence for some reason

Reply:
I could have been clearer too, or just made the change suggestion to begin with. 😄